### PR TITLE
Removed #nullable disabled

### PR DESCRIPTION
**Pull request by stefannikolei**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/SixLabors/ImageSharp/pulls) open
- [x] I have verified that I am following the existing coding patterns and practice as demonstrated in the repository. These follow strict Stylecop rules :cop:.
- [x] I have provided test coverage for my change (where applicable)

### Description
Removed `#nullable disable` I all files where nearly the only problem was in the Equals method

#2231

### CODE REVIEW

File: src/ImageSharp/Metadata/Profiles/ICC/TagDataEntries/IccNamedColor2TagDataEntry.cs [hunk 1]
```diff
@@ -72,7 +71,7 @@ public IccNamedColor2TagDataEntry(string prefix, string suffix, IccNamedColor[]
     /// <param name="suffix">Suffix</param>
     /// <param name="colors">The named colors</param>
     /// <param name="tagSignature">Tag Signature</param>
-    public IccNamedColor2TagDataEntry(int vendorFlags, string prefix, string suffix, IccNamedColor[] colors, IccProfileTag tagSignature)
+    public IccNamedColor2TagDataEntry(int vendorFlags, string? prefix, string? suffix, IccNamedColor[] colors, IccProfileTag tagSignature)
```
Comment (JimBobSquarePants) [inline src/ImageSharp/Metadata/Profiles/ICC/TagDataEntries/IccNamedColor2TagDataEntry.cs:74]: What's going on here? When would prefix/suffix be null?

Comment (stefannikolei) [reply src/ImageSharp/Metadata/Profiles/ICC/TagDataEntries/IccNamedColor2TagDataEntry.cs:74]: Have a look at L:51

Comment (JimBobSquarePants) [reply src/ImageSharp/Metadata/Profiles/ICC/TagDataEntries/IccNamedColor2TagDataEntry.cs:74]: Ahah Thanks! I didn't see cos of the collapsed regions. All good 👍

**Review by JimBobSquarePants — APPROVED** [review]
Looks good to me. Thanks!

**Pull request merged**